pkg/kube/secret/secret.go

```diff
@@ -133,3 +133,19 @@ func EnsureSecretWithKey(secretGetUpdateCreateDeleter GetUpdateCreateDeleter, ns
 	}
 	return string(existingSecret.Data[key]), nil
 }
+
+// CopySecret copies secret object(data) from one cluster to another
```

Comment:
[q] looks like this can be used also to copy a secret, let's say, from one namespace to another in the same cluster by passing the same client in the first two parameters, right? If so can we update the comment?

@tibulca if this gets merged before you finish your PR this might interest you.

Reply:
>  looks like this can be used also to copy a secret, let's say, from one namespace to another in the same cluster by passing the same client in the first two parameters
 
 yes, will update the comment.